src/test/resources/org/jenkinsci/plugins/plaincredentials/SecretBytesTest/loadUnencrypted/config.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson/>
```

Comment:
well the tests were failing.... perhaps I just need to upgrade the test harness in the `pom.xml` to fix that... but I'll leave that as an exercise for a future PR